mix.exs

```diff
@@ -65,7 +65,12 @@ defmodule Explorer.MixProject do
         "Functions: Multi-table": &(&1[:type] == :multi),
         "Functions: Row-based": &(&1[:type] == :rows),
         "Functions: Introspection": &(&1[:type] == :introspection),
-        "Functions: IO": &(&1[:type] == :io)
+        "Functions: IO": &(&1[:type] == :io),
+        # For series
+        "Functions: Aggregation": &(&1[:type] == :aggregation),
+        "Functions: Element-wise": &(&1[:type] == :element_wise),
+        "Functions: Transformation": &(&1[:type] == :transformation),
+        "Functions: Window": &(&1[:type] == :window)
```

Comment:
We are missing introspection! Generate the docs and make sure that “Functions” subcategory does not appear in the Series page.

Reply:
Introspection was already in use by the `DataFrame` module, so I didn't add it.